include/drivers/adp536x.h

```diff
@@ -4,19 +4,182 @@
  * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
  */
 
+#ifndef ADP536X_H_
+#define ADP536X_H_
+
+/**@file adp536x.h
+ *
+ * @brief Driver for ADP536X.
+ * @defgroup adp536x
+ * @{
+ */
+
 #include <zephyr.h>
 
+#define ADP536X_VBUS_ILIM_50mA  0x00
```

Comment:
Are these aligned with the rest of the definitions locally? If not, it would be nice to have all at same indent. I know Github can sometimes garble the formatting, so might be fine.

Reply:
Fixed.